tests/e2e/test_bot_picklability.py

```
@@ -17,7 +17,7 @@ def test_pickle_bot_attributes():
     import bots.tools.self_tools
     import bots.tools.terminal_tools
 
-    bot = AnthropicBot(model_engine="claude-haiku-4-5-20251001", max_tokens=1000, temperature=0.0)
+    bot = AnthropicBot(model_engine=Engines.CLAUDE45_HAIKU, max_tokens=1000, temperature=0.0)
```

Ruff (0.15.9)

[error] Undefined name `Engines`

(F821)